packages/test-interface-criteria/tsconfig.gen.json

```diff
@@ -0,0 +1,16 @@
+/* For emitting .d.ts files, we use a separate tsconfig file, because we do not
+want to emit .d.ts files for unit tests */
```

Comment:
This was messing with the emitted types, making them not work, hence the issue https://github.com/openactive/openactive-test-suite/issues/676